include/sisl/cache/lru_evictor.hpp

```diff
@@ -67,10 +67,6 @@ class LRUEvictor : public Evictor {
 
     public:
         LRUPartition() = default;
-        LRUPartition(const LRUPartition&) = delete;
-        LRUPartition& operator=(const LRUPartition&) = delete;
-        LRUPartition(LRUPartition&&) = default;
-        LRUPartition& operator=(LRUPartition&&) = default;
```

Comment:
I've let the compiler determine this synthetically; having `std::mutex` will automatically delete assignments operators etc anyways.